Procfile

```diff
@@ -1,3 +1,3 @@
-web: bundle exec passenger start -p $PORT -a 0.0.0.0
-worker: bundle exec sidekiq --queue app_perf --verbose
+web: bundle exec rails s -p 5000
```

Comment:
Does the port need to be hard coded here?

Reply:
It doesn't need to be hard coded, so it can be using `-p $PORT`

Reply:
Right, but isn't that what it should be here, so that someone can set `PORT`?